Corrected BaseCommand.check() signature in docs. (#17764)
django/django: docs/howto/custom-management-commands.txt

    It may return a string which will be printed to ``stdout`` (wrapped
    by ``BEGIN;`` and ``COMMIT;`` if :attr:`output_transaction` is ``True``).

.. method:: BaseCommand.check(app_configs=None, tags=None, display_num_errors=False, include_deployment_checks=False, fail_level=checks.ERROR, databases=None)

    Uses the system check framework to inspect the entire Django project for
    potential problems. Serious problems are raised as a :exc:`CommandError`;
    warnings are output to ``stderr``; minor notifications are output to
    ``stdout``.

    If ``app_configs`` and ``tags`` are both ``None``, all system checks are

>     performed except deployment checks which default to False. ``tags`` can be a list of check tags, like ``compatibility`` or

Reviewer: ```suggestion
    performed except deployment and database related checks. ``tags`` can be a
    list of check tags, like ``compatibility`` or ``models``.
```

>     ``models``. Database checks will be skipped with ``databases`` set to the default None.

Author: Hey @felixxm should I change the entire instruction for the new suggestion or should it just be an addition?

>     ``models``. Database checks will be skipped with ``databases`` set to the default None.

Reviewer: I'd [add](https://github.com/django/django/pull/17764/files#r1461448061) _"and database related checks"_ to the existing description, and a [new sentence](https://github.com/django/django/pull/17764/files#r1461453497) about `include_deployment_checks` and `databases` below.

>     ``models``. Database checks will be skipped with ``databases`` set to the default None.

Author: Completed
